Runtime/Core/CinemachineDebug.cs

```diff
@@ -17,6 +17,7 @@ public static void ReleaseScreenPos(Object client)
                 mClients.Remove(client);
         }
 
+#if CINEMACHINE_UNITY_IMGUI
```

Comment:
It's strange to remove GetScreenPos but leave in ReleaseStringPos.  They should either both be in or both be out.